sw/device/tests/pmp_sanitytest_napot.c

```diff
@@ -0,0 +1,110 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/runtime/pmp.h"
+
+#include "sw/device/lib/handler.h"
+#include "sw/device/lib/irq.h"
+#include "sw/device/lib/runtime/check.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/test_main.h"
+#include "sw/device/lib/testing/test_status.h"
+
+#define PMP_LOAD_REGION_ID 0
```

Comment:
Some of these also don't need to be macros. Please convert them to constants in a followup.

Reply:
I will do this in a follow-up.